flytekit/core/python_function_task.py

```diff
@@ -112,6 +112,7 @@ def __init__(
         node_dependency_hints: Optional[
             Iterable[Union["PythonFunctionTask", "_annotated_launch_plan.LaunchPlan", WorkflowBase]]
         ] = None,
+        pickle_untyped: bool = False,
```

Comment:
@kumare3 Since proposed the `safe` parameter name, are you okay with using `pickle_untyped`?

My concerned is that `safe=False` could mean so many unsafe behavior. `pickle_untyped` is explicit about the behavior.